### DISCUSSION TITLE
vite构建下使用FloatingBubble组件,编译报错

### DISCUSSION THREAD

**Discussion by oh-my-zack** [youzan/vant#12026] [Q&A]
node: 18.16.1,
vue: ^3.2.37,
vite: 3.2.7,
vant: 4.6.0
![image](https://github.com/youzan/vant/assets/131418801/36a72d70-633f-43d1-b47f-26ab562b80dc)

**Comment by zhousg** [comment]
请确认vant版本，如果无误删除 node_modules 再试试， 如果不行可否在提供其他更具体打包环境信息和参考代码？
